### ISSUE TITLE
Crash related to map / apply; crash on world load

### ISSUE THREAD

**Issue by neoc666** [CyclopsMC/IntegratedDynamics#570]
IntegratedDynamics-1.12.2-0.11.10
IntegratedTunnels-1.12.2-1.5.5

I'm playing FTBRevelation-1.8.0-1.12.2 , added Integrated Dynamics and Tunnels manually, disabled a lot of mods.

I had multiple crashes when inserting a (map (apply arith-equals 10) list) into a display. Each time my client crashed and after reloading the world the display and the map card were gone. Then I did the same with a (map (apply add 10) list). Again my client crashed. Now my world is corrupted and I can't reload it.

Seems to be related to #262 and #265.

Log for crash when trying to load corrupted world (again): https://pastebin.com/zAaUAbmX
Log for crash when trying to load corrupted world (first time): https://pastebin.com/qQWysXyi
Log for crash immediately before that: https://pastebin.com/FwdzqmLA

**Comment by rubensworks** [comment]
Did you also `flip` something? The crashlog seems to suggest so.

(Note that you can start ID in safe-mode (config file) to recover your world)

**Comment by neoc666** [comment]
You mean flip arguments to an operator? I tried something like that before, but I always cleared the variable cards when I was done with an experiment.

**Comment by rubensworks** [comment]
Do you have any details on that? Because that seems to have caused the crash.

**Comment by neoc666** [comment]
One time after I crashed I had the idea that one of the two-argument-operators expected the arguments in the opposite order. So I enclosed something in a flip. I don't remember the details though, because when I now think about it, I have no idea which operator I enclosed in a flip - I'd guess it must have been one with two different argument types. But then it couldn't have been arithmetic-equals (for the 'filter list by 10'-quest) or arithmetic-addition, because those take two of the same type.

Btw, was able to recover my world with safe-mode, thanks for that!

**Comment by neoc666** [comment]
I just tried again in my recovered world (I loaded in safe-mode, broke all ID blocks, restarted without safe-mode).

Am I doing it the right way if I want to map a list with a +3 operation? I made the following stuff:

1. A list [1, 10, 100]
2. An integer 3
3. An operator arithmetic addition
4. An apply of 3) to 2)
5. A map from 4) to 1)
6. I put 1), 2), 3), 4) into a variable store connected to a display via cable
7. I put 5) into the display.

Then I crashed. Log: 
[latest.zip](https://github.com/CyclopsMC/IntegratedDynamics/files/1937176/latest.zip)
